lib/pure/collections/tables.nim

```diff
@@ -1068,22 +1068,25 @@ proc largest*[A](t: CountTable[A]): tuple[key: A, val: int] =
   result.key = t.data[maxIdx].key
   result.val = t.data[maxIdx].val
 
-proc sort*[A](t: var CountTable[A]) =
+proc sort*[A](t: var CountTable[A], descending: bool = true) =
```

Comment:
Maybe instead of a `bool` flag reuse the `SortOrder` enum from `algorithm`?

Reply:
@Vindaar Ah, I didn't know about that enum! Thanks, I agree that's a better choice. I'll revise the pull request.

Reply:
Now using SortOrder instead of a boolean. Thanks again for this good suggestion.